pkgs/applications/video/kodi/build-kodi-binary-addon.nix

```
@@ -0,0 +1,31 @@
+{ stdenv, toKodiAddon, addonDir, cmake, kodi, kodi-platform, libcec_platform }:
+{ name ? "${attrs.pname}-${attrs.version}"
```

```suggestion
{ pname ? "${attrs.pname}
, version ? "${attrs.version}"
```
I think this should be done for all addons to make name not ambiguous.

---

Copy/paste of the `python` module. I assume the flexibility is "worth it". Will leave as is.